app.py

```diff
@@ -1,8 +1,11 @@
 import os
 from flask import Flask
 from api import api
+from flask_cors import CORS
 
 app = Flask(__name__)
+CORS(app)
```

Comment:
It'd probably be better to do something like `CORS(app, origins='*')`. While this _should_ be equivalent to what you had it makes it more obvious that one can restrict the domain from where access is allowed.

Reply:
Agreed. I can make that change and add a CORS_ENABLE flag to `config.py`... unless it be preferable to have that be specified elsewhere.

BTW, as per @bdwyer2 suggestion I submitted a similar PR over in `codait/max-base` but will continue here until we reach a solution

Reply:
See https://github.com/IBM/MAX-Object-Detector, which already does something similar for the embedded web app.

Reply:
Changed to match environment variable pattern from https://github.com/IBM/MAX-Object-Detector

`CORS_ENABLE=true` is now specified in the `run` command, e.g. `docker run -e CORS_ENABLE=true`

Tested/confirmed that it is blocked by default but allowed when flag is set.